### PR TITLE
Feature/open virtual keyboard

### PR DESCRIPTION
**Pull request by nirmalbaldaniya-dream**

### CODE REVIEW

File: openVirtualKeyboard.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * 
+ * Usage pattern:
+ *  - Import 
+ *    - import { openVirtualKeyboard } from '@dreamworld/web-util.js'
+ *  - Use
+ *    - openVirtualKeyboard();
+ *
+ */
+
+class OpenVirtualKeyboard {
```
Comment (ChiragMoradiya) [inline openVirtualKeyboard.js:14]: Here, we don't need to create a class, it's redundant from my point of view and makes this logic hard to understand.

Comment (ChiragMoradiya) [inline openVirtualKeyboard.js:14]: We don't use camel-case in the file name ever. It's always all small letters separated by dash.

**Comment by ChiragMoradiya** [comment]
I have made changes to this source, please review it and test it again. Please note, you need to integrate loglevel here.

**Comment by ChetanGoti** [comment]
I have created a new branch for this and released it as well. 


> Published release 1.4.0-open-virtual-keyboard-new.1 on open-virtual-keyboard-new channel

**Pull request closed without merging**